DEVEL.md

```diff
@@ -0,0 +1,78 @@
+# Developing the Maistra Istio Operator
+
+## Building the binary and the container image
+
+To build the Operator container image, run:
+```bash
+IMAGE=docker.io/maistra/istio-operator:latest tmp/build/docker_build.sh
```

Comment:
Can we use other image name? Like `docker.io/<your-username>/istio-operator:latest`

Reply:
I didn't want to use `<your username>`, but couldn't think of a good example username (best I could think of was `johndoe`), so I went with `maistra` - I thought it would be obvious people should replace this with their own usernames... but now I agree `<your username>` is perfectly fine.

Anyway, I've fixed it now.